### PR TITLE
Update profile fields

### PR DESCRIPTION
**Pull request by thuphan1312**
Update profile characteristics with the ones used for search API

### CODE REVIEW

**Review by angelikaladia — COMMENTED** [review]
Few fixes but mostly good.

File: src/client/js/profile.js [hunk 1]
```diff
@@ -22,8 +24,9 @@ export class ProfileView {
     //Future iteration will pull from db
     this.#profilePhoto = "./img/example.jpg";
     this.#name = "Angelika Ladia";
-    this.#zipcode = "08854";
-    this.#preferredLength = "4 mi";
+    this.#latitude = "120";
+    this.#longitude = "100";
+    this.#preferredtrailtype = "Easy";
```
[the remark this answers is not in the document]
Comment (dominicrutk) [reply src/client/js/profile.js:29]: To clarify a little, the display names should be capitalized, and the "values" of the select options should be lowercase, exactly as in `search.js`. For instance, one of the options should appear as "Mountain Biking" to the user, and its underlying value should be "mountain biking".

File: src/client/js/profile.js [hunk 2]
```diff
@@ -140,35 +149,40 @@ export class ProfileView {
...
   #name = "Angelika";
-  // Zipcode
-  #zipcode = "08846";
-  // Preferred length
-  #preferredLength = "4 mi";
+  // Lat and Lon
+  #lat = "120";
+  #lon = "200";
+  // Preferred type
+  #preferredType = "Easy";
```
Comment (dominicrutk) [inline src/client/js/profile.js:185]: Same as @angelikaladia's comments about: lat and lon should be numbers, and preferred type should be a trail type, not a difficulty

File: src/client/js/profile.js [hunk 3]
```diff
@@ -256,17 +276,21 @@ class EditProfileView {
...
 
-  /** Edit preferred length
+  /** Edit preferred type
    * @param {string} new_preferredLength - The new preferred length to be saved
    */
-  #editPreferredLength(new_preferredLength) {
-    this.#preferredLength = new_preferredLength;
+  #editPreferredType(new_preferredType) {
+    this.#preferredType = new_preferredType;
   }
```
Comment (dominicrutk) [inline src/client/js/profile.js:295]: Update these comments and add a comment for `#editLon`

**Comment by thuphan1312** [comment]
Fixed the fields based on Review

**Review by angelikaladia — APPROVED** [review]

**Pull request merged**